src/main/java/org/apache/sling/commons/log/logback/internal/config/LoggerManagedServiceFactory.java

```diff
@@ -20,17 +20,23 @@
 
 import java.util.Dictionary;
 
+import org.apache.sling.commons.log.logback.internal.LogConfigManager;
 import org.osgi.service.cm.ManagedServiceFactory;
 
 class LoggerManagedServiceFactory extends LogConfigurator implements ManagedServiceFactory {
 
+    public static final String LOG_FILE_DEFAULT = "logs/error.log";
+
     public String getName() {
         return "Logger configurator";
     }
 
     public void updated(String pid, @SuppressWarnings("rawtypes") Dictionary configuration)
             throws org.osgi.service.cm.ConfigurationException {
         try {
+            if (configuration.get(LogConfigManager.LOG_FILE) == null) {
```

Comment:
We should not assume that the configuration dictionary is mutable

Reply:
@cziegeler  please recheck